server/app/controllers/kommission-controller.js

```diff
@@ -0,0 +1,114 @@
+const kommissionService = require('../services/kommission-service');
+const { Kommission } = require('../models/kommission');
+const { checkStructure, createUserError } = require('../../utils');
+
+/**
+ * Fetch all the kommissions from the database.
+ *
+ * @param req Request
+ * @param res Response
+ * @return {Promise.<void>} Nothing
+ */
+async function getAllKommissions(req, res) {
+    const kommissions = await kommissionService.getAllKommissions();
+
+    res.json(kommissions);
+}
+
+/**
+ * Create a kommission.
+ *
+ * @param req Request
+ * @param res Response
+ * @return {Promise.<void>} Nothing
+ */
+async function createKommission(req, res) {
+    
+    // FIXME We should check the type of each provided field, instead of just the presence
+    if (!checkStructure(req.body, ['id', 'name', 'description'])) {
+        throw createUserError(
+            'BadRequest',
+            'The body has missing properties, needed: [\'id\', \'name\', \'description\']'
+        );
+    }
+    
+    let newKommission = new Kommission({
+        id: req.body.id,
```

Comment:
Because the field `id` is auto-generated, you don't want to let an user specify the _Kommission's id_